packages/luna/src/components/PlanCard/PlanCard.tsx

```diff
@@ -0,0 +1,206 @@
+            </Typography>
+          </div>
+        ) : null}
+      </div>
+      <Typography
+        element="tiny-text"
+        weight="semibold"
+        className={cn(
+          "tw-inline-block tw-mb-4",
+          primary ? "tw-text-natural-100" : "tw-text-natural-700"
+        )}
+      >
+        {description}
+      </Typography>
+      <Typography
+        element="h4"
+        weight="bold"
+        className={cn(
+          "tw-mb-2",
+          primary ? "tw-text-natural-50" : "tw-text-brand-700"
+        )}
+      >
+        {intl("plan.weekly-minutes", { value: formatNumber(weeklyMinutes) })}
+      </Typography>
+      <div className="tw-flex tw-gap-2 tw-items-end tw-flex-wrap">
+        <Typography
+          element="subtitle-1"
+          weight="bold"
+          className={cn(primary ? "tw-text-natural-50" : "tw-text-natural-950")}
+        >
+          {formatNumber(priceAfterDiscount, {
+            maximumFractionDigits: 2,
+          })}
+        </Typography>
+        <Typography
+          element="subtitle-1"
+          weight="bold"
+          className={cn(primary ? "tw-text-natural-50" : "tw-text-natural-950")}
+        >
+          {intl("labels.currency.egp")}
+        </Typography>
+        {discount ? (
+          <div className="tw-flex tw-gap-1">
+            <Typography
+              element="tiny-text"
+              weight="regular"
+              className={cn(
+                primary ? "tw-text-natural-100" : "tw-text-natural-700"
+              )}
+            >
+              {intl("plan.instead-of")}
+            </Typography>
+            <Typography
+              element="tiny-text"
+              weight="regular"
+              className={cn(
+                "tw-block tw-relative",
+                primary ? "tw-text-destructive-400" : "tw-text-destructive-600"
+              )}
+            >
+              {intl("plan.price-before-discount", {
+                value: formatNumber(price),
+              })}
+              <span
+                className={cn(
+                  "tw-absolute tw-block tw-top-1/2 tw-right-0 tw-left-0 tw-border-t",
+                  primary
+                    ? "tw-border-destructive-400"
+                    : "tw-border-destructive-600"
+                )}
+              />
+            </Typography>
+          </div>
+        ) : null}
+      </div>
+      {primary ? (
+        <button
+          type="button"
+          className={cn(
+            "tw-w-full tw-py-3 tw-rounded-lg tw-mt-6",
+            "tw-bg-natural-50 hover:tw-bg-brand-100 active:tw-bg-brand-200"
+          )}
+          onClick={onBuy}
+        >
+          <Typography
+            element="body"
+            weight="semibold"
+            className={cn(primary ? "tw-text-brand-700" : "tw-text-brand-50")}
+          >
+            {intl("plan.button.buy")}
+          </Typography>
+        </button>
+      ) : (
+        <Button size={ButtonSize.Small} className="tw-mt-6 tw-w-full">
```

Comment:
You forget to add the onclick event on this button as well.

Reply:
Also, add the button type to be button.


```suggestion
        <Button size={ButtonSize.Small} htmlType="button" className="tw-mt-6 tw-w-full">
```